exports/context.tf

```
@@ -54,6 +54,8 @@ variable "context" {
     regex_replace_chars = string
     label_order         = list(string)
     id_length_limit     = number
+    label_value_case    = string
+    label_key_case      = string
```

Traditionally and alphabetically, key comes before value
```suggestion
    label_key_case      = string
    label_value_case    = string
```